scripts/emulator_launch_app.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+source ./scripts/emulator_read_target.sh
+
+timeout_seconds=1800  # default timeout set to 30 minutes
+
+function check_metro_server() {
+    metro_status=$(cat ./${output_dir}/react-native_start.txt | grep Metro)
```

Comment:
path could be wrong if script is not run from root directory. solve with `git rev-parse --show-toplevel` or error if not in root